vue.config.js

```diff
@@ -41,41 +40,63 @@ module.exports = {
       app.use(bodyParser.urlencoded({ extended: true }))
       app.use(bodyParser.json())
 
-      // Mock a route to mimic this upload result format:
-      // {
-      //   'audio_file_id': 147,
-      //   'duration_seconds': '4.803250',
-      //   'description': 'xyz.wav',
-      //   'created_at': {
-      //     'date': '2020-11-24 01:41:58',
-      //     'timezone_type': 3,
-      //     'timezone': 'UTC'
-      //   },
-      //   'audio_uuid': '5fbc64e0c74e90.82972899'
-      // }
-      app.all('/audiofiles/upload', (req, res) => {
+      /**
+       * Mock the flowjs-2.0.0 backend GET route to mimic the upload result format:
+       * {
+       *   'audio_file_id': 147,
+       *   'duration_seconds': '4.803250',
+       *   'description': 'xyz.wav',
+       *   'created_at': {
+       *     'date': '2020-11-24 01:41:58',
+       *     'timezone_type': 3,
+       *     'timezone': 'UTC'
+       *   },
+       *   'audio_uuid': '5fbc64e0c74e90.82972899',
+       *   'uri': 'some_uri_value',
+       * }
+       */
+      app.get('/audiofiles/upload', (req, res) => {
+        // Simulate `unknown upload chunk` with 404 status for GET method
+        res.writeHead(404, { 'Content-Type': 'application/json' })
+        res.end(JSON.stringify({status: 'error', status_description: 'Unknown upload chunk'}))
+      })
+      /**
+       * Mock the flowjs-2.0.0 backend POST route
+       * Note, flowjs sends form-data as payload
+       */
+      app.post('/audiofiles/upload', bodyParser.json(), (req, res) => {
         const now = new Date()
           .toISOString()
           .split('T')
 
+        const audio_uuid = `${Math.random()
+          .toString(36)
+          .substr(2, 16)}.${Math.random()
+          .toString(36)
+          .substr(2, 10)}`
+        // TODO (Nice to have): replace original_extension with `req.body.flowFilename.split('.').pop()` when we know how to parse form-data from POST request
```

Comment:
I spent 3h to use `req.body` when we pass form-data from frontend, but didn't success. For now, I'm putting this TODO here as Nice to have. It is not worth spending much more hours I think.